src/main.rs

```diff
@@ -1467,6 +1478,7 @@ fn main() {
             create.culture(matches.value_of("culture"));
             create.debug_build(matches.is_present("debug-build"));
             create.debug_name(matches.is_present("debug-name"));
+            create.extension(matches.value_of("extension"));
```

Comment:
See earlier comment about changing `extension` to `file-extension` to avoid confusion.